cmd/notary/prettyprint.go

```diff
@@ -198,6 +198,19 @@ func prettyPrintRoles(rs []*data.Role, writer io.Writer, roleType string) {
 	table.Render()
 }
 
+// Pretty-prints a list of delegation paths, and ensures the empty string is printed as "" in the console
+func prettyPrintPaths(paths []string) string {
+	prettyPaths := []string{}
+	for _, path := range paths {
+		// manually escape "" and designate that it is all paths with an extra print <all paths>
+		if path == "" {
+			path = "\"\" <all paths>"
```

Comment:
I like @cyli's suggestion. Sort in lexical order, maybe we can even add some output syntax to indicate shadowing?